src/elli/importer/spectraray.py

```diff
@@ -8,48 +8,69 @@
 
 
 def read_spectraray_psi_delta(
-    fname: str, sep: str = r"\s+", decimal: str = "."
+    fname: str, sep: str = " ", decimal: str = "."
```

Comment:
Why not use any whitespace as separator? My gut feeling is that it's a little bit more resilient against formatting errors in files. Or did you encounter errors with `\s+`?

Reply:
Ah, the problem is not the raw string, but rather that str.split does not evaluate regexes. You could use `re.split`. I'll put a suggestion below